chef/data_bags/crowbar/template-provisioner.json

```diff
@@ -9,6 +9,7 @@
       "access_keys": "",
       "shell_prompt": "USER@HOST:CWD SUFFIX",
       "enable_pxe": true,
+      "self_update_url": "http://crowbar/sle12/$arch",
```

Comment:
I would change the default to: `http://<ADMINWEB>/suse-12.2/$arch/repos/installer-update` (or something similar).

A couple of things:
- `<ADMINWEB>` should be substituted in the rails app (see glance & tempest barclamps)
- if we put `suse-12.2` in the path, then maybe that attribute should live in a versioned suse attribute path somewhere (we already have that in the provisioner barclamp, I think)
- we need a doc bug to document how to mirror that repo in the location